tasks/notify.py

```diff
@@ -400,6 +404,21 @@ def run_impl_within_lock(
             )
             return {"notified": False, "notifications": None}
 
+    def is_using_codecov_commenter(
+        self, repository_service: TorngitBaseAdapter
+    ) -> bool:
+        """Returns a boolean indicating if the message will be sent by codecov-commenter.
+        If the user doesn't have an installation, and if the token type for the repo is codecov-commenter,
+        then it's likely that they're using the commenter bot.
+        """
+        commenter_bot_token = get_config(repository_service.service, "bots", "comment")
+        return (
+            repository_service.service == "github"
+            and repository_service.data.get("installation") is None
+            and repository_service.get_token_by_type(TokenType.comment)
+            == commenter_bot_token
```

Comment:
I'm not 100% sure about this part. Am I missing some checks to make sure that the user doesn't have any installations and is also using the commenter bot?

Reply:
This looks OK

Maybe just for peace of mind you should add 1 extra check `commenter_bot_token is not None` to make sure something is configured for the commenter bot...